netsim/ansible/templates/initial/cumulus_nvue.j2

```diff
@@ -19,40 +60,11 @@
 {% endif %}
 {% if 'ipv6' in loopback %}
             {{ loopback.ipv6 }}: {}
+{% else %}
+          ipv6:
+            enable: off
 {% endif %}
         type: loopback
-{% for l in interfaces|default([]) if l.type in ['lan','p2p','stub','svi'] %}
-      {{ l.ifname }}:
-        link:
-{%  if l.mtu is defined %}
-          mtu: {{ l.mtu }}
-{%  endif %}
-          state:
-            up: {}
-        type: {{ 'svi' if l.type=='svi' else 'swp' }}
-{%   if l.name is defined %}
-        description: "{{ l.name }}{{ " ["+l.role+"]" if l.role is defined else "" }}"
-{%   elif l.type|default("") == "stub" %}
-        description:  Stub interface
-{%   endif %}
-{%   if (l.ipv4 is defined and l.ipv4) or l.ipv6 is string %}
-        ip:
-          address:
-{%     if l.ipv4 is defined %}
-{%      if l.ipv4 == True %}
-            {{ loopback.ipv4 }}: {}
-{%       else %}
-            {{ l.ipv4 }}: {}
-{%       endif %}
-{%     endif %}
-{%     if l.vrf is defined %}
-          vrf: {{ l.vrf }}
-{%     endif %}
-{%     if l.ipv6 is defined %}
-{%       if l.ipv6 == True %}
-{%       else %}
-            {{ l.ipv6 }}: {}
-{%       endif %}
-{%     endif %}
-{%   endif %}
+{% for l in interfaces|default([]) if l.type in ['lan','p2p','stub','svi','lag'] %}
```

Comment:
Why do you need to check interface type here? What interfaces do you not want to configure and why?

Reply:
It’s a historic leftover, “loopback” type comes to mind (though that would be caught elsewhere)

I can submit another PR to add loopback support and remove (test) this check on type

Reply:
Please do. Thank you!